digitalfabrik/entitlementcard
# 1313: Expose android ports

### Short description

As a developer i have to expose the ports for local android builds manually.

### Proposed changes



- expose android ports for local builds if android device is connected

Fixes: #1313

### Testing:
- connect android device
- run local build
- stores should be loaded on map and search
- if no android device is connected the build should also start (ios build)
